### DISCUSSION TITLE
Microsoft Store (new) intune applications

### DISCUSSION THREAD

**Discussion by Nouneds** [Micke-K/IntuneManagement#204] [General]
Hello @Micke-K 
I wonder if you are able to help me find a part of code that is responsible for retrieving application list from tenant. 
The thing is that the Intune Manager gets a whole response with Microsoft Store (new) apps included and when i try to retrieve those through different ps commands or even get method from microsoft graph explorer they are not listed. Wonder what is done differently.

Kind regards

**Comment by Micke-K** [comment]
Hello,

This is API call I use in Intune Manager. This should work in Graph Explorer as well.

`/beta/deviceAppManagement/mobileApps?$filter=(microsoft.graph.managedApp/appAvailability eq null or microsoft.graph.managedApp/appAvailability eq 'lineOfBusiness' or isAssigned eq true)&$orderby=displayName`

Let me know how it goes. I also handle paging so all apps are returned.  If you have a lot of apps it might only return first page without the Microsoft Store (New) apps. Check for '@odata.nextLink' value in the return. If that is set you need to call the url in that property to get the next page.

Cheers!

**Reply by Nouneds** [reply]
Hi,
that worked well, thank you!!
